lib/src/css_printer.dart

```diff
@@ -201,101 +262,106 @@ class CssPrinter extends Visitor {
 
     if (_isTesting) {
       // Emit assuming url() was parsed; most suite tests use url function.
-      emit(' @import url(${node.import})');
+      emit('@import url(${node.import})');
     } else if (isStartingQuote(node.import)) {
-      emit(' @import ${node.import}');
+      emit('@import ${node.import}');
     } else {
       // url(...) isn't needed only a URI can follow an @import directive; emit
       // url as a string.
-      emit(' @import "${node.import}"');
+      emit('@import "${node.import}"');
     }
     emitMediaQueries(node.mediaQueries);
-    emit(';');
+    _emitSemicolon(forceLf: true);
   }
 
   @override
   void visitKeyFrameDirective(KeyFrameDirective node) {
-    emit('$_newLine${node.keyFrameName} ');
+    emit('${node.keyFrameName} ');
     node.name!.visit(this);
-    emit('$_sp{$_newLine');
+    emit(_sp);
+    _emitLBrace();
     _isInKeyframes = true;
     for (final block in node._blocks) {
       block.visit(this);
     }
     _isInKeyframes = false;
-    emit('}');
+    _emitRBrace();
   }
 
   @override
   void visitFontFaceDirective(FontFaceDirective node) {
-    emit('$_newLine@font-face ');
-    emit('$_sp{$_newLine');
+    emit('@font-face ');
+    emit(_sp);
```

Comment:
I think there's an extra/unnecessary space here. I'm guessing this was unintentional in the original code.

Reply:
ah, yup; I'll remove one of these